extendeddatacrossword_test.go

```diff
@@ -110,3 +112,45 @@ func TestRepairExtendedDataSquare(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkRepair(b *testing.B) {
+	// For different ODS sizes
+	for i := 16; i <= 128; i *= 2 {
+		for _codecType := range codecs {
+			// Generate a new range original data square then extend it
+			square := genRandDS(i)
+			eds, err := ComputeExtendedDataSquare(square, _codecType, NewDefaultTree)
+			if err != nil {
+				b.Error(err)
+			}
+
+			flattened := eds.flattened()
+			// Randomly remove half the shares
+			for i := 0; i < i*i*2; {
+				ind := rand.Intn(i)
+				if len(flattened[ind]) == 0 {
+					continue
+				}
+				flattened[ind] = []byte{}
+				i++
+			}
```

Comment:
It's not too important I don't think, but this doesn't guaranteed delete half the data, it could delete less.

Reply:
I agree it's not too important if it doesn't exactly delete half the shares (so long as it's not more than half). But why would it delete less?

Reply:
whoops, you're right and I'm blind

Reply:
I had actually moved the `i++` into the for loop, then realized that, no, it does in fact belong where it is. 😂